backstop/tests/table.html

```diff
@@ -1847,1876 +2040,4005 @@ <h2 id="regular">Zebra striping</h2>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 2"
+              type="button"
+            >
+              <span class="iui-label">2</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 3"
+              type="button"
+            >
+              <span class="iui-label">3</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 4"
+              type="button"
+            >
+              <span class="iui-label">4</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 5"
+              type="button"
+            >
+              <span class="iui-label">5</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 6"
+              type="button"
+            >
+              <span class="iui-label">6</span>
+            </button>
+
+            <button
+              class="iui-button iui-borderless iui-small"
+              aria-label="Page 7"
+              type="button"
+            >
+              <span class="iui-label">7</span>
+            </button>
+
+            <span class="iui-ellipsis iui-small">&mldr;</span>
+
+            <div class="iui-progress-indicator-radial iui-determinate iui-small">
+              <svg
+                class="iui-radial"
+                viewBox="0 0 40 40"
+                aria-hidden="true"
+              >
+                <circle
+                  class="iui-track"
+                  cx="20"
+                  cy="20"
+                  r="15.91549"
+                >
+                </circle>
+                <circle
+                  class="iui-fill"
+                  cx="20"
+                  cy="20"
+                  r="15.91549"
+                  style="stroke-dashoffset: 20;"
+                >
+                </circle>
+              </svg>
+            </div>
+          </span>
+
+          <button
+            class="iui-button iui-borderless iui-small"
+            aria-label="Next page"
+            type="button"
+          >
+            <!-- Use chevron-right.svg from @itwin/itwinui-icons package. -->
+            <svg
+              class="iui-icon"
+              aria-hidden="true"
+              viewBox="0 0 16 16"
+            >
+              <path d="m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z" />
+            </svg>
+          </button>
+        </div>
+
+        <div class="iui-right">
+          <label class="iui-input-container">
+            <div class="iui-select">
+              <div
+                class="iui-select-button"
+                tabindex="0"
+              >1-10 of many</div>
```

Comment:
"many" sounds scary (also needs extra localization). Wonder if there's a better way to display uncertainty

Reply:
The **of many** should only show up while the system is still calculating the total amount of pages.  It should _hopefully_ only be for a couple of seconds.  But I am open to suggestions on what else to display.

Reply:
does it really need to change the display for 2 seconds? The loading spinner should be enough.
It could just show the "old" number of items and then update when the calculation is done. Also could disable the button for that duration maybe.

Reply:
I'll change it to only show **1-10 of...**.  Once the counting is done, we'd replace the **...** with the item count.

Reply:
How much work would it be to do an animated version of `...`?